src/components/popup/WalletHeader.tsx

```diff
@@ -242,18 +234,23 @@ export default function WalletHeader() {
         }
       }
     ];
-    if (!isExpanded && items.length === 4) {
+
+    if (location.pathname === "/popup.html") {
```

Comment:
The full screen version is now loaded from a different HTML file: `fullscreen.html`, so all these checks are now much simpler and the styling changes require no JS.